shared/middleware/helpers/bucket_helper.rb

```diff
@@ -260,7 +262,7 @@ def list_versions(encrypted_channel_id, filename)
 
   # Copies the given version of the file to make it the current revision.
   # (All intermediate versions are preserved.)
-  def restore_previous_version(encrypted_channel_id, filename, version_id)
+  def restore_previous_version(encrypted_channel_id, filename, version_id, user_id = nil)
```

Comment:
Is there a reason for the default value on this method but not the other? My first thought was that we don't want to break other callers, but I only see the one call to this method.

Reply:
yes, but not a good one :trollface:  fixed.